src/launchpad/client.ts

```diff
@@ -63,18 +69,23 @@ export class LaunchpadClient {
         headers["x-api-key"] = this.api_key;
       }
 
+      // Choose the base URL based on whether L402 is used or not
+      const baseURL = options?.useL402
+        ? "https://ordinalsbot.ln.sulu.sh/runes/launchpad/"
+        : this.env === "live"
+          ? "https://api.ordinalsbot.com/runes/launchpad/"
+          : "https://testnet-api.ordinalsbot.com/runes/launchpad/";
+
+      // Create the Axios client with the appropriate base URL
```

Comment:
This is not runes/launchpad, it should be:
          this.env === "live"
            ? `https://api.ordinalsbot.com/launchpad/`
            : `https://testnet-api.ordinalsbot.com/launchpad/`,